src/main/java/no/sikt/sws/ListIndecesHandler.java

```diff
@@ -0,0 +1,52 @@
+package no.sikt.sws;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import no.sikt.sws.exception.SearchException;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ListIndecesHandler extends ApiGatewayHandler<String, String> {
+
+    public static final String RESOURCE_IDENTIFIER = "resource";
```

Comment:
Is this used? If so, I dont belive it should be - it specifies the index-name in the other handler

Reply:
agreed